app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -51,11 +51,25 @@ class PatientSearchResultItemView(
     renderPatientNameAgeAndGender(searchQuery, model)
     renderPatientAddress(model.address)
     renderPatientDateOfBirth(model.dateOfBirth)
-    renderPatientPhoneNumber(model.phoneNumber)
+    val number = highlightPatientPhoneNumber(searchQuery, model)
+    renderPatientPhoneNumber(number)
     renderVisited(model.lastSeen)
     renderLastSeen(model.lastSeen, currentFacilityId)
   }
 
+  private fun highlightPatientPhoneNumber(searchQuery: String?, patientSearchResult: PatientSearchResultViewModel): PhoneNumber {
```

Comment:
Same comment as other highlight function, let's name this `patientPhoneNumber` since it can return a plain phone number as well.